app/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,17 @@
+<resources>
+    <string name="app_name">Food Expiration Dates</string>
+    <string name="app_name_short">Food Expirations</string>
```

Comment:
Could you translate also these strings?

For the string `app_name`, please add the corresponding translation, and for the string `app_name_short`, if there is a short way to translate (as in English), add it, but if there's not and the translation is equal to `app_name`, consider to link the two string as follows:
```
<string name="app_name_short">@string/app_name</string>
```

Reply:
Done. Please take a look in the updates